.github/workflows/py-deb-workflow.yml

```diff
@@ -154,7 +158,7 @@ jobs:
     needs: [Build]
     runs-on: ubuntu-20.04
     container:
-      image: registry.aws.cloud.mov.ai/devops/ros-buildtools-noetic:latest
+      image: registry.aws.cloud.mov.ai/qa/ros-buildtools-noetic:latest
```

Comment:
`devops`, `qa`, `develop`, `prod` we have too many namespaces I think, one should be enough

Reply:
yep true.